resource/reactivedrop_koreana.txt

```diff
@@ -611,8 +611,8 @@
 "rd_objective_chat_message"		" 목표 %s1  완료! \n시간: %s2  이전 목표로부터 소요된 시간: %s3\n"
 "[english]rd_points_awarding_message"		"Awarding %s4%s1 points to %s4%s2, total = %s4%s3\n"
 "rd_points_awarding_message"		"%s4%s2 해병이 %s4%s1 점 획득, 합계 점수 = %s4%s3\n"
-"[english]rd_points_reduced_due_to_farming"		"You recently played this map. Gained points will be decreased.\n"
-"rd_points_reduced_due_to_farming"		"해당 맵을 플레이한지 얼마 되지 않아 획득 점수가 감소하였습니다.\n"
+"[english]rd_points_reduced_due_to_farming"		"You recently played this map. Gained points have been decreased.\n"
+"rd_points_reduced_due_to_farming"		"최근에 이 맵을 플레이했습니다. 획득한 점수가 감소했습니다.\n"
```

Comment:
최근 이 맵을 플레이해서 획득 점수가 감소했습니다.

I think I can write it more concisely. :)

Reply:
Hmm, that's cleaner with fewer characters, I like that better.

Reply:
> Hmm, that's cleaner with fewer characters, I like that better. /글자수 적어서 더 좋네요.

아무래도 클리어 시 나오는 문구들이 많다 보니 화면을 가리는 공간이 많지 않나 싶었습니다.

Reply:
I don't have time to play games these days, so I don't know how the lettering works in-game, and if there were fewer characters, that would be good because it would block less of the screen.
/
요즘 게임할 시간이 없어서 게임 내에서 어떤 방식으로 글자가 나오는지 제가 그걸 몰라요. 글자 수가 더 적다면 화면을 덜 가릴테니 그게 좋겠죠.